sig-service-catalog/charter.md

```diff
@@ -0,0 +1,166 @@
+  - See [OWNERS](https://raw.githubusercontent.com/kubernetes-incubator/service-catalog/master/OWNERS) for who has access.
+- [Image Repository](https://quay.io/organization/kubernetes-service-catalog)
+  - Canary builds are published on pushes to master.
+  - Release builds (and latest) are published on tags.
+  - Chairs have access to manage this repository.
+- [Helm Repository](https://svc-catalog-charts.storage.googleapis.com)
+  - Charts are manually published after each release.
+  - Managed by Vic Iglesias (Google), @viglesias on the kubernetes slack.
+- [svc-cat.io](https://svc-cat.io)
+  - Published on pushes to master.
+  - Site hosted with [Netlify](https://app.netlify.com/sites/svc-cat/overview).
+  - Chairs and interested maintainers have access to manage this site.
+- [CLI Binary Hosting](https://svc-cat.io/docs/install/#manual)
+  - Canary builds are published on pushes to master.
+  - Release builds (and latest) are published on tags.
+  - Files hosted on Azure blob storage.
+  - Azure account managed by Carolyn Van Slyck (Microsoft) and Aaron Schlesinger
+    (Microsoft).
+- [Travis](https://travis-ci.org/kubernetes-incubator/service-catalog)
+  - Runs the CI builds.
+  - Maintainers have access.
+- [Jenkins](https://service-catalog-jenkins.appspot.com/)
+  - Runs end-to-end tests on a live cluster.
+  - Server managed by Michael Kibbe (Google).
+
+## Roles
+
+- Maintainers
+  - Maintainer is equivalent to the standard [Kubernetes definition of
+    Approver](https://github.com/kubernetes/community/blob/master/community-membership.md#approver).
+  - Responsible for reviewing pull requests, and approving pull requests for merge.
+  - Responsible for technical planning and stewardship of the project.
+  - New maintainers may be nominated by another maintainer, and accepted via lazy
+    two-thirds resolution amongst the chairs.
+  - Maintainers may be nominated for removal from their position by a chair,
+    and accepted via lazy two-thirds resolution amongst the chairs.
+  - Maintainers may propose changes to this charter at any time, by submitting a
+    pull request and then notifying the SIG mailing list, to be accepted via
+    lazy two-thirds resolution amongst the chairs.
+
+- Chairs
+  - Chairs are expected to perform the role of maintainer, in addition to their chair responsibilities.
+  - Chairs are listed in our [SIG
+    definition](https://github.com/kubernetes/community/tree/master/sig-service-catalog#chairs).
+  - Responsible for project administration activities within the SIG and are
+    non-technical in nature, such as organizing the weekly meetings.
+  - A chair does not have more rights, or votes, than a maintainer.
+  - Responsible for reporting the SIG’s status to the appropriate Kubernetes
+    leadership teams.
+  - All decisions amongst chairs are made using lazy consensus with a fallback
+    to a 2/3 majority vote (lazy two-thirds resolution).
```

Comment:
(non-blocking): are there are a minimum number of chairs?

Reply:
We talked about it but we didn't feel the need to have a specific number. There is a line about that further down in the section

https://github.com/kubernetes/community/pull/2233/files#diff-d62d482f9a1782544ed6617b746e7ca2R92

`There are no set number of Chairs`